source/VSTSExtensions/OctopusBuildAndReleaseTasks/Tasks/Push/Octopus-Push.ps1

```diff
@@ -7,12 +7,19 @@ try {
 
     . .\Octopus-VSTS.ps1
 
+    $IgnorePullRequestMergeBuilds = Get-VstsInput -Name IgnorePullRequestMergeBuilds -AsBool
     $OctoConnectedServiceName = Get-VstsInput -Name OctoConnectedServiceName
     $ConnectedServiceName = Get-VstsInput -Name ConnectedServiceName
     $Packages = Get-VstsInput -Name Package -Require
     $AdditionalArguments = Get-VstsInput -Name AdditionalArguments
     $Replace = Get-VstsInput -Name Replace -AsBool
 
+    if( ${env:BUILD_SOURCEBRANCH}  -match "refs/pull/.*/merge$" -and [System.Convert]::ToBoolean($IgnorePullRequestMergeBuilds))
```

Comment:
Will `$IgnorePullRequestMergeBuilds` already be a boolean due to the `-AsBool` above?